craft_store/errors.py

```diff
@@ -64,3 +69,10 @@ def __init__(self, response: requests.Response) -> None:
             "Issue encountered while processing your request: "
             f"[{response.status_code}] {response.reason}."
         )
+
+
+class NotLoggedIn(CraftStoreError):
+    """Error raised when credentials are not found in the keyring."""
+
+    def __init__(self):
+        super().__init__("Not logged in.")
```

Comment:
This is why I want the message to be passed on the point of exception. "Not logged in" looks too generic.

Reply:
updated in https://github.com/canonical/craft-store/pull/5/commits/08d4f17ff5bd3b188a57c0befcf9e85312ccdc58